sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -27,6 +27,20 @@
 import AzureCore
 import Foundation
 
+internal class StorageJSONDecoder: JSONDecoder {
```

Comment:
Needed to use these in BlobTransfer so I moved them out, though they remain internal.

Reply:
Sounds good, but maybe we should put them in a separate file then.

Reply:
Sounds good, but maybe we should put them in a separate file then.

Reply:
Done. Placed in `Util/StorageJSONCoding.swift`